src/cmds/testing/thread/thread_priority_test.c

```diff
@@ -0,0 +1,82 @@
+/**
+ * @file
+ *
+ * @brief Testing interface for thread with different priority
+ *
+ * @date 2021.12.31
+ * @author: kpishere
+ */
+
+#include <embox/test.h>
+
+#include <kernel/thread.h>
+#include <kernel/time/ktime.h>
+#include <kernel/task.h>
+#include <kernel/task/kernel_task.h>
+#include <framework/mod/options.h>
+
+#include <pthread.h>
+
+#include <util/err.h>
+#include <util/log.h>
```

Comment:
Please use usual printf instead of log_xxx in commands we try that a command can be compiled in Linux environment